iceoryx_hoofs/include/iceoryx_hoofs/cxx/string.hpp

```diff
@@ -580,12 +588,14 @@ class string
     /// @note out of bounds access leads to program termination
     constexpr const char& operator[](const uint64_t pos) const noexcept;
 
+    // AXIVION Next Construct AutosarC++19_03-A11.3.1: It's Ok
```

Comment:
For the justification you could state that:
- it's a template class
- the string needs to be constructible from a string with another capacity, comparable with another string, etc.
- this doesn't break the encapsulation

Reply:
@FerdinandSpitzschnueffler Sure.